site/app/Resources/views/base.html.twig

```diff
@@ -155,6 +150,12 @@
       <script src="{{ asset_url }}"></script>
     {% endjavascripts %}
 
+    {{ encore_entry_script_tags('app') }}
+
+    <!-- Renders app.js & a webpack runtime.js file -->
+    <script src="/build/runtime.js"></script>
```

Comment:
If you are using `{[ encore_entry_script_tags('...') }}` then you do not need explicit `script` tags. The macro will render it for you.